src/mongocrypt-kms-ctx.c

```diff
@@ -1384,9 +1566,18 @@ _mongocrypt_kms_ctx_init_gcp_decrypt (
    char *request_string;
    bool ret = false;
 
+   if (!kms) {
```

Comment:
Suggest converting into a not-null assertion.

Reply:
Changed as suggested.  Will show up in next commit.